src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,46 @@
+.clock {
+  display: flex;
+
+  position: relative;
+
+  align-items: center;
+  justify-content: center;
+
+  @include clock-circle;
+
+  background-color: #bb8f6a;
+
+  &__minutes {
+    width: $clock-minute;
+    height: 5px;
+
+    @include clock-hands-position;
+
+    background-color: black;
+
+    animation: move 3600s linear infinite;
+  }
+
+  &__seconds {
+    width: $clock-second;
+    height: 3px;
+
+    @include clock-hands-position;
+
+    background-color: red;
+
+    animation: move 60s linear infinite;
+  }
```

Comment:
This part of the code is repeated for two arrows so it will be good to create a mixin with some arguments for each arrow.